src/AggregateSpecsPlugin.ts

```diff
@@ -72,15 +75,15 @@ export const PgAggregatesSpecsPlugin: GraphileConfig.Plugin = {
           },
           fallback: PgCodec<any, any, any, any>
         ) => {
-          return (
+          return EXPORTABLE((dataTypeToAggregateTypeMap, fallback) => (
             codec: PgCodec<any, any, any, any>
-          ): PgCodec<any, any, any, any> => {
+          ): any => {
```

Comment:
Why did this become any? I think `PgCodec` has 7 parameters now, so maybe this needs to be `PgCodec<any, any, any, any ,any, any, any>`?

Reply:
Ah, missed that. Debugging an issue and forgot to put that back. Fixed!